problems/kattis/quadrant/java/jasonBengtson/Quadrant.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+
+public class Main {
+
+    static void Quad(double x, double y) {
+        if(x > 0 && y > 0)
+            System.out.println("1");
+        else if(x < 0 && y > 0)
+            System.out.println("2");
+        else if(x < 0 && y < 0)
+            System.out.println("3");
+        else if(x > 0 && y < 0)
+            System.out.println("4");
+        else
+            System.out.println("Incorrect input");
+
+    }
+
+
+    public static void main(String[] args) {
+        double x,y;
+
+        Scanner s = new Scanner(System.in);
+        System.out.println("Enter your x value: ");
+        x = s.nextDouble();
+        System.out.println("Enter your y value: ");
+        y = s.nextDouble();
+        System.out.println("You entered x = " + x + " and y = " + y);
+
+        Quad(x, y);
+
+
+
+    }
+
+
+
+
+

```

Comment:
Ok man I copied 
this and ran it in my IDE. +10 as long as you fixed your fork problem

![jason](https://user-images.githubusercontent.com/42912657/75301462-1be94600-57f8-11ea-9a1f-0244a0726293.JPG)

Reply:
Thanks!